src/main/scala/mesosphere/marathon/core/task/jobs/impl/OverdueInstancesActor.scala

```diff
@@ -65,12 +65,12 @@ private[jobs] object OverdueInstancesActor {
 
       def launchedAndExpired(task: Task): Boolean = {
         task.status.condition match {
-          case Condition.Provisioned | Condition.Starting if task.status.stagedAt < unconfirmedExpire =>
+          case Condition.Provisioned | Condition.Staging if task.status.stagedAt < unconfirmedExpire =>
```

Comment:
I don't understand: `taskLaunchConfirmationTimeout` is defined as `Time, in milliseconds, to wait for a task to enter the TASK_STAGING state before killing it`. But in this line, we'll fail if `Condition.Staging` is achieved? Shouldn't it be only `Condition.Provisioned`?

Reply:
Good point